docs/user-guide/index.md

```diff
@@ -1,17 +1,14 @@
-# Getting Started
+# Browsertrix User Guide
 
-## Signup
+Welcome to the Browsertrix user guide! While our aim with our software is to create interfaces so intuitive no additional explination is needed, sometimes the complexities of web archiving require a little more expansion. If there's something that you found especially confusing or frustrating and — after searching for it here — if no satisfactory answer was given, we'd like to know about it, [please get in touch](docs-feedback@webrecorder.net)!
```

Comment:
oop TY for spelling fix!

Didn't like the double "explanation", hence "expansion".  If you've got something better swap it in!  Otherwise will commit this, much better!

Reply:
how about "While our aim is to create intuitive interfaces, sometimes the complexities of web archiving require a little more explanation"?

Reply:
Great!